package.json

```diff
@@ -16,11 +16,11 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@swc/core": "^1.2.30"
+    "@swc/core": "*"
   },
   "devDependencies": {
-    "@swc/core": "^1.2.77",
-    "@types/node": "^14.11.1",
+    "@swc/core": "*",
```

Comment:
I'm not sure about this. Why is `*` used?

Reply:
I thought the @swc/core should keep the latest version, but here, manually upgrade it is better~. I will revert this change